tools/java/org/unicode/cldr/util/DtdData.java

```diff
@@ -1210,6 +1210,10 @@ public AttributeStatus getAttributeStatus(String elementName, String attributeNa
             if (elementName.startsWith("icu:")) {
                 return AttributeStatus.distinguished;
             }
+            if ("grammaticalState".equals(elementName)) {
+                // ??? TODO: "Dtd supplementalData doesn’t allow element=grammaticalState, attribute: values"
```

Comment:
Something odd here. I don't find grammaticalState in any dtd; might be mis-generated.

Reply:
I'll try to repro this and provide details.

Reply:
I can't repro anymore, so I'll restore the original code.